crates/pipeline-manager/src/api/support_data_collector.rs

```diff
@@ -223,6 +224,7 @@ impl SupportBundleData {
             version: 2,
             time: chrono::Utc::now(),
             circuit_profile: Ok(vec![1, 2, 3]),
+            json_circuit_profile: Ok(vec![1, 2, 3]),
```

Comment:
maybe we should increase `current_versoin` not strictly necessary for adding a field but if it turns out to be a problem it's easier to distinguish the bundles later